src/middlewares/upload.js

```diff
@@ -0,0 +1,14 @@
+import multer from 'multer';
```

Comment:
what does this file do ?

Reply:
it allows using form/multipath while sending the request so that we can get files

Reply:
uploading file

Reply:
Thanks @alainburindi, I think that is clearer than my explanation